apps/nerves_hub_www/assets/js/dates.js

```diff
@@ -9,11 +11,11 @@ let formatDateTime = datetime => {
     .split(' ')
     .join('T')
 
-  if (datetime == 'never' || datetime == '') {
+  if (datetime === 'never' || datetime === '') {
     return datetime
   } else {
     const date = new Date(datetime)
-    return date.toLocaleString('en-US', { timeZoneName: 'short' })
+    return moment(date).format('MMM Do, YYYY [at] h:mma')
```

Comment:
I think you'll want to use moment here to convert to the local timezone. All the dates coming back from the DB should be GMT/UTC.

Reply:
https://momentjs.com/docs/#/parsing/
> `moment(...)` is local mode. Ambiguous input (without offset) is assumed to be local time. Unambiguous input (with offset) is adjusted to local time.

There's no difference when `.local()` is used. https://momentjs.com/docs/#/manipulating/local/

Reply:
This is why I was banging my head for a bit:


> Also note that if your original input contains a time zone offset such as +00:00 or Z, then you can just parse it directly with moment. You don't need to use .utc or .local. For example:
`var localDate = moment("2015-01-30T10:00:00Z");`

https://stackoverflow.com/questions/29220467/how-to-convert-moment-js-date-to-users-local-timezone

And why I was thinking it could be a local issue?

Reply:
YOU'RE RIGHT